src/components/action-bar/action-bar-item/action-bar-item.tsx

```diff
@@ -0,0 +1,123 @@
+import { Component, Event, EventEmitter, h, Prop } from '@stencil/core';
+import { ActionBarItem } from '../action-bar.types';
+import { createRandomString } from '../../../util/random-string';
+import { ListSeparator } from 'src/interface';
+
+/**
+ * @private
+ */
+@Component({
+    tag: 'limel-action-bar-item',
+    shadow: false,
+    styleUrl: 'action-bar-item.scss',
+})
+export class ActionBarButton {
+    /**
+     * Item that is placed in the action bar.
+     */
+    @Prop()
+    public item!: ActionBarItem | ListSeparator;
+
+    /**
+     * Fired when a action bar item has been clicked.
+     */
+    @Event()
+    public select: EventEmitter<ActionBarItem | ListSeparator>;
+
+    /**
+     * Used to attach the right tooltip to the right button
+     */
+    private tooltipId: string;
+
+    constructor() {
+        this.tooltipId = createRandomString();
+    }
+
+    public render() {
+        return this.renderButton(this.handleClick);
+    }
+
+    private renderButton(handleClick: (event: MouseEvent) => void) {
+        if (!this.isItem(this.item) && this.item.separator) {
+            return <div role="separator" />;
+        }
+
+        return (
+            <button
+                id={this.tooltipId}
+                type="button"
+                onClick={handleClick}
+                disabled={this.isDisabled()}
+            >
+                {this.renderIcon()}
+                {this.renderLabel()}
+                {this.renderTooltip()}
```

Comment:
Do we really need the tooltip? It's a bit annoying
<img width="502" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/5563232/71ba35f6-af12-4d0a-8bd3-eafdc1c45721">

Reply:
yes we need. for reasons that I don't want to explain 😄 the tooltip has a [shitty disgusting bug](https://github.com/Lundalogik/lime-elements/issues/1408) that we never have fixed. This is why it becomes annoying. We should fix that bug instead. 😞 I can't, otherwise I'd have done it 2 years ago

Reply:
ok! 🙂